src/pages/_playground/api-explorer.tsx

```diff
@@ -21,9 +21,18 @@ export async function getStaticProps() {
     }
   }
 
-  const params = queries.getParams().addInclude(['field_cta'])
```

Comment:
curious if this got pushed in on accident or if you wanted to update

Reply:
probably don't need to update it here, will revert